analyses/telomerase-activity-prediction/02-Comparing-Counts-versus-FPKM.R

```diff
@@ -0,0 +1,77 @@
+library('pracma')
```

Comment:
I think most of these packages are not needed - I successfully ran this script loading _only_ two: packages `ggpubr` and `grid`. Try and see how many of these library statements can be removed to get the minimal set of packages needed.